spec/ruby/language/predefined_spec.rb

```diff
@@ -879,6 +879,23 @@ def obj.foo2; yield; end
   end
 end
 
+describe "Global variable $VERBOSE" do
+  it "mostly allows truthy values" do
```

Comment:
Why `mostly`?

Reply:
I think something like `converts truthy values to true` would be more descriptive of the behavior (and the false case would need to be split then).

Reply:
I think the name "truthy" has come to mean handling both `true` and `false` cases, which is why I said mostly. But your suggestion sounds clearer.